main.cpp

```diff
@@ -13,75 +13,193 @@ What you create in this project will be used as the basis of Project 5 in the co
 */
+
+/*
+
+
+
+
+
+/*
+Thing 1) dictionary
+5 properties:
+    1) amount of ink used
+    2) number of pages
+    3) edition of book
+    4) date of publication
+    5) weight of book
+3 things it can do:
+    1) store word definitions
+    2) weight another object down
+    3) provide synonyms for a word
+ */
+
+/*
+Thing 2) Airplane
+5 properties:
+    1) number of seats
+    2) number of crewmembers
+    3) length in miles to destination
+    4) amount of fuel consumed per flight
+    5) number of passengers per trip
+3 things it can do:
+    1) transport passengers
+    2) feed passengers
+    3) fly at high altitudes
+ */
+
+/*
+Thing 3) production company
+5 properties:
+    1) amount of money fundraised
+    2) amout of money spent on advertising per year
+    3) number of employees
+    4) amount of revenue per year
+    5) number of cameras
+3 things it can do:
+    1) produce movies every year
+    2) employ technical artists
+    3) partner with entertainment company
+ */
+
+/*
+Thing 4) drumset
+5 properties:
+    1) number of cymbals
+    2) age of kit in years
+    3) number of pieces
+    4) snare drum diameter
+    5) ride cymbal stand height
+3 things it can do:
+    1) break a drumhead 
+    2) play music loudly (db)
+    3) adjust the stool height
+ */
+
+/*
+Thing 5) oscillator
+5 properties:
+    1) frequency control (hz)
+    2) waveform bank
+    3) pulse-width control
+    4) FM input
+    5) amplitude level
+3 things it can do:
+    1) produce an audible tone
+    2) modulate another parameter at audio rates
+    3) change timbre
+ */
+
+/*
+Thing 6) filter
+5 properties:
+    1) cutoff frequency (hz)
+    2) slope (db)
+    3) filter bank
+    4) resonance amount
+    5) drive amount
+3 things it can do:
+    1) subtract harmonics from an oscillator
+    2) self-oscillate
+    3) modulate the cutoff frequency
+ */
+
+/*
+Thing 7) envelope
+5 properties:
+    1) attack time (ms)
+    2) hold time (ms)
+    3) decay time (ms)
+    4) sustain level (midi)
+    5) release time (ms)
+3 things it can do:
+    1) shape the amplitude of an oscillator
+    2) modulate other parameters with positive values
+    3) play in one-shot mode, or cycle
+ */
+
+/*
+Thing 8) LFO
+5 properties:
+    1) frequency control (hz)
+    2) waveform bank
+    3) phase shifter
+    4) sync to tempo button
+    5) offset control
+3 things it can do:
+    1) modulate other synth parameters with positive and negative values
+    2) oscillate at audio rates
+    3) 
+ */
+
+/*
+Thing 9) Sequencer
+5 properties:
+    1) number of tracks
+    2) number of steps
+    3) tempo control (bpm)
+    4) glide control (ms)
+    5) quantizer control (midi)
+3 things it can do:
+    1) pattern melodic sequences
+    2) arpeggiate chords
+    3) pattern drum beats
+ */
+
+/*
+Thing 10) Synthesizer
+5 properties:
+    1) Oscillator
+    2) Filter
+    3) Envelope
+    4) Low Frequency Oscillator
+    5) Sequencer
+3 things it can do:
```

Comment:
A `Low Frequency Oscillator` is still an Oscillator.
you need to come up with a different object for 4). 
perhaps an `FX` section? 
- [ ] revise